src/components/Join.js

```diff
@@ -61,7 +61,7 @@ const Consent = compose(
   }) => {
     return (
       <ConsentContainer>
-        <h3 className={theme.h3}>
+        <h3 css={`${theme.h3}`}>
```

Comment:
I've got a comment near the bottom discussing this pattern, I think we should just add proper `P`, `Hx` components to the uikit so we don't need to go through `css` / `theme` every time